_security/configuration/configuration.md

```diff
@@ -79,7 +79,7 @@ These are the allowed values for `type`:
 
 - `basic`: HTTP basic authentication. No additional configuration is needed.
 - `kerberos`: Kerberos authentication. Additional, [Kerberos-specific configuration](#kerberos) is needed.
```

Comment:
Do you mean "Additionally,"?

Reply:
Apparently the original writer was on a deadline. removed comma.